src/views/GovernanceReviewTeam/Actions/ExtendLifecycleId.tsx

```diff
@@ -131,14 +144,18 @@ const ExtendLifecycleId = ({
           notificationRecipients
         }
       }
-    }).then(response => {
-      if (!response.errors) {
-        history.push(`/governance-review-team/${systemId}/notes`);
-        onSubmit();
-      }
     });
+
+    // If no errors, view intake action notes
+    if (!response.errors) {
+      history.push(`/governance-review-team/${systemId}/notes`);
+      onSubmit();
+    }
   };
```

Comment:
I think there could be error handling here if the mutation fails to extend the lifecycleID